include/dt-bindings/gpio/gpio.h

```diff
@@ -1,180 +1,453 @@
 /*
  * Copyright (c) 2018 Linaro Limited
+ * Copyright (c) 2018 Peter Bigot Consulting, LLC
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 #ifndef ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_GPIO_H_
 #define ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_GPIO_H_
 
+/*
+ * Assigned Bit Positions
+ *
+ * @note: This table is intended as a convenience to summarize the bit
+ * assignments used in the fields defined below.  The GPIO_foo_SHIFT
+ * macros are the canonical definition.
+ *
+ * Bit   Prefix         Variations
+ * ===== ============== =======================================================
+ * 0     GPIO_ACTIVE    high or low
+ * 1     GPIO_SIGNALING single-ended or push-pull
+ * 2     GPIO_LINE      source or drain
+ * 3-4   GPIO_PUD       pull normal/up/down
+ * 5-8   GPIO_IO        direction in/out, set output value
```

Comment:
Because direction is a sub-component of GPIO_IO.  There's no conflict.  A requested new feature was to specify that pins configured as output be given an initial value.  So GPIO_IO combines direction and, when output, the initial value, just as the table documents.